IceCreamShop.playground/Contents.swift

```diff
@@ -1 +1,80 @@
 
+import UIKit
+
+struct Flavor {
+    var name: String
+    var rating: Double
+}
+
+
+enum Size: Double {
+    case small = 3.99
+    case medium = 4.99
+    case large = 5.99
+}
+ 
+struct Cone {
+    var flavors: Flavor
+    var size: Size
+    var topping: String?
+    
+    
+    func eat(){
+        print("Mmm! I love \(flavors.name) ice cream!")
+    }
+    
+}
+
+class IceCreamShop {
+    var allFlavors: [Flavor]
+    var sizes: Size
+    var toppings: [String]
+    var totalSales: Double
+    
+    init(allFlavors: [Flavor], sizes: Size, toppings: [String], totalSales: Double) {
+        self.allFlavors = allFlavors
+        self.sizes = sizes
+        self.toppings = toppings
+        self.totalSales = totalSales
+}
+    
+    func listTopFlavors() {
+        var message: String = "Our top flavors are: "
+        
+        for flavor in allFlavors {
+            if flavor.rating >= 4 {
+                print(message += flavor.name)
+            }
+        }
+    print(message)
+    }
```

Comment:
The () () that this is printing out I actually find very odd as I would expect it to print the top flavors 1 by 1 instead of () (). I may need to ask Spencer why it is printing like that...
```suggestion
    func listTopFlavors() {
        var message: String = "Our top flavors are: "
        
        for flavor in allFlavors {
            if flavor.rating >= 4 {
                message += flavor.name
            }
        }
    print(message)
    }
```
If you have it like this, just removing the print statement in the for loop it doesn't print those () ()